### PR TITLE
Move the pipeline to ADO to allow runs from fork

### PR DESCRIPTION
**Pull request by manvkaur**
Move the PR and CI build to ADO to allow runs from forks with a comment from maintainer

Possible conflict with #30

### CODE REVIEW

File: build/build-pr-ci.yml
Comment (cgillum) [inline build/build-pr-ci.yml:33]: This should be 8.x. Generally speaking, we should avoid depending on STS releases when possible.

**Review by cgillum — COMMENTED** [review]
LGTM. I have one suggestion below. Also, I think we should wait to merge this after merging @aishwaryabh's PR.

File: build/build-pr-ci.yml
Comment (manvkaur) [reply build/build-pr-ci.yml]: Yes, it should have been 8, updated.

**Review by cgillum — APPROVED** [review]

**Comment by manvkaur** [comment]
Addressing https://github.com/Azure/azure-functions-pyfx-planning/issues/195

**Pull request merged**
